.clang-format

```diff
@@ -0,0 +1,66 @@
+# Generated from CLion C/C++ Code Style settings
```

Comment:
Please use the clang-format in fcitx5 repo

https://github.com/fcitx/fcitx5/blob/master/.clang-format

Be sure to return clang-format after change it.